### PR TITLE
[W04] Add edgeCase test to MedianPivotDistributed

### PR DESCRIPTION
**Pull request by fabiomanz**

### CODE REVIEW

File: w04/test/gad/simplesort/MedianPivotDistributedTest.java [hunk 1]
```diff
@@ -69,4 +69,75 @@ private static Stream<Arguments> fromEquals0() {
...
+                        0,
+                        2,
+                        0,
+                        1),
+                Arguments.of(
+                        new int[]{9},
+                        0,
+                        1,
+                        0,
+                        0),
```
Comment (nilsreichardt) [inline w04/test/gad/simplesort/MedianPivotDistributedTest.java]: Thanks for your tests! I'm failing these two cases but have 100% of the points on this exercise.

Comment (fabiomanz) [reply w04/test/gad/simplesort/MedianPivotDistributedTest.java]: Does it work for you when you change the `numbersOfConsideredElements` to 3 or 5?
I sadly cannot reproduce it as my implementation also works with these values.

Comment (nilsreichardt) [reply w04/test/gad/simplesort/MedianPivotDistributedTest.java]: Yes, with `3` and `5` is works for me 👍

Comment (fabiomanz) [reply w04/test/gad/simplesort/MedianPivotDistributedTest.java]: I updated the branch

**Review by nilsreichardt — APPROVED** [review]
LGTM

**Pull request merged**